cloud/wfm-rest-api/src/impl/MongoDbRepository.ts

```diff
@@ -13,7 +14,10 @@ const dbError: ApiError = new ApiError(errorCodes.DB_ERROR, 'MongoDbRepository d
 /**
  * Service for performing data operations on mongodb database
  */
-export class MongoDbRepository<T extends { id: string }> implements PagingDataRepository<T> {
+export class MongoDbRepository<T extends {
+  id: string,
+  _id?: string | ObjectID
```

Comment:
I think we have two options. Move all queries to use _id. or remove _id completely from every query.
I think I also used wrong syntax for update https://stackoverflow.com/questions/22356110/update-field-inside-document-by-id-in-mongodb

Another thing will be to grab that obj into interface. This looks really strange